tests/storage/export.go

```diff
@@ -203,7 +203,9 @@ var _ = SIGDescribe("Export", func() {
 		} else {
 			addFilesystemVolume(pod, volumeName)
 		}
-		return libpod.RunPod(pod)
+		pod, err := libpod.Run(pod, testsuite.GetTestNamespace(pod))
+		Expect(err).ToNot(HaveOccurred())
+		return pod
```

Comment:
I hope that in a future PR, the `err` can be properly returned instead of being swallowed here. It is hard to debug failed `Expect`s that are hidden inside functions.